DEVELOPING.md

```diff
@@ -354,6 +354,36 @@ stacktrace that was saved as a txt file.
 $ ndk-stack -sym obj/arm64-v8a -dump trace.txt
 ```
 
+## Release management
+
+The Mapbox Maps SDK for Android makes stable releases every 6 weeks. Beta build is available in week 2 and a release candidate in week 4, which is promoted to stable in week 6.
```

Comment:
Add a not that we treat stable release as new major or minor release in semver terms?

Reply:
added this to semver section